dist/tcti-tabrmd.pc.in

```diff
@@ -1,7 +1,7 @@
 Name: tcti-tabrmd
 Description: TCTI library for communicating with the TPM2 access broker / resource manager daemon (tabrmd).
-URL: https://github.com/01org/??
+URL: https://github.com/01org/tpm2-abrmd
 Version: @VERSION@
-Requires: sapi
+Requires: sapi glib-2.0 gio-2.0
```

Comment:
Sorry I didn't understand the question. Dump the file list for libglib2.0-dev package on your system. It has both glib and gio libraries (like you say) but it also has a separate '.pc' file for each. Since the TCTI library depends on both I've included both in it's '.pc' file.